signs-work-order-map/components/ControlledNumberInput.tsx

```diff
@@ -0,0 +1,37 @@
+import Form from "react-bootstrap/Form";
+import { Controller, Control } from "react-hook-form";
+import { CalculationInputsType } from "@/types/calcs";
+
+interface ControlledNumberInputProps {
+  name: keyof CalculationInputsType;
+  control: Control<CalculationInputsType>;
+  valueMaximum: number;
+  label: string;
+}
+
+export default function ControlledNumberInput({
+  name,
+  control,
+  valueMaximum,
+  label,
+}: ControlledNumberInputProps) {
+  return (
+    <Controller
+      control={control}
+      name={name}
+      render={({ field: { onChange, value } }) => (
+        <div className="d-flex justify-content-between">
+          <Form.Label className="col-3">{label}</Form.Label>
+          <Form.Control
+            type="number"
```

Comment:
We should consider using the `inputmode="numeric` [pattern](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/) here. I'm not sure how relevant these issues still are, but we went that route in the VZE.